alliance/apps/backlog/views/list_backlogs_view.py

```diff
@@ -76,14 +80,11 @@ def post(self, request):
         return JsonResponse(results)
 
     def update_estimate(self, request, results, team_id):
-        backlog_id = request.POST.get('estimate-backlog_id')
         from ...backlog.forms import EstimateForm
-        try:
-            estimate = Estimate.objects.get(
-                team_id=team_id, backlog_id=backlog_id)
-        except Estimate.DoesNotExist:
-            estimate = None
 
+        backlog_id = request.POST.get('estimate-backlog_id')
+        estimate = Estimate.objects.get_object_or_none(
```

Comment:
Same comment here, `get_object_or_none` is a function on its own, not a method of `objects`.